### ISSUE TITLE
Media Library not uploading after insert of existing media

### ISSUE THREAD

**Issue by stuartshields** [humanmade/altis-media#84]
So there is a bug currently, that when you attempt to upload an image after inserting existing media into a post. The upload fails (with console error).

Steps to replicate:
1. Create new post
2. Insert media from existing library
3. Upload image into media library
4. View browser console to see error.

Console error *:
```
Uncaught TypeError: this.$index is undefined
    info https://example-stag.altis.cloud/wp-includes/js/media-views.min.js?ver=5.3.4:1
    Backbone 11
    u https://example-stag.altis.cloud/wp-includes/js/plupload/wp-plupload.min.js?ver=5.3.4:1
    forEach Underscore
    u https://example-stag.altis.cloud/wp-includes/js/plupload/wp-plupload.min.js?ver=5.3.4:1
    dispatchEvent https://example-stag.altis.cloud/wp-includes/js/plupload/plupload.min.js?ver=2.1.9:1
    trigger https://example-stag.altis.cloud/wp-includes/js/plupload/moxie.min.js?ver=1.3.5:1
    addFile https://example-stag.altis.cloud/wp-includes/js/plupload/plupload.min.js?ver=2.1.9:1
    t https://example-stag.altis.cloud/wp-includes/js/plupload/moxie.min.js?ver=1.3.5:1
```

I've confirmed this happening on two different client sites on both Altis v3 and Altis v4

* https://example-stag.altis.cloud/ is used instead because of clients name.

**Comment by roborourke** [comment]
In some situation there's a `ready` event or callback on a subview that isn't being fired. I can't work out why at the moment. How often are people hitting this situation?

**Comment by roborourke** [comment]
Fixed in Smart Media 0.2.19, you can upgrade by running `composer update altis/media --with-dependencies`. Please reopen if you see the problem persist.

**Comment by stuartshields** [comment]
@roborourke I don't think this is fixed. I've done the update and the error still happens.

To reproduce:
1. Select existing media into image block
2. try and upload a new image to a new image block

Image fails to upload (same error as reported before).

**Comment by stuartshields** [comment]
@roborourke as per slack. I cloned down `smart-media` removed `trigger` and replaced it with `render` but the issue still occurs after `npm run build`.

**Comment by roborourke** [comment]
Fixed in smart media 0.2.21, for real this time.
